backend/backend/settings/common.py

```diff
@@ -215,4 +215,11 @@ def to_bool(value):
 
 GZIP_MODELS = to_bool(os.environ.get('GZIP_MODELS', False))
 
+STATSD_ENABLED = to_bool(os.environ.get('STATSD_ENABLED', False))
+STATSD_HOST = os.environ.get('STATSD_HOST', 'localhost')
+STATSD_PORT = os.environ.get('STATSD_PORT', 8125)
```

Comment:
`os.environ` return string, so you should cast in int if this what you need :)

Reply:
done